From 7b8e72409cd134d044d28682247aabf377c9b5a8 Mon Sep 17 00:00:00 2001
From: HeavyTony2 <97621927+HeavyTony2@users.noreply.github.com>
Date: Thu, 8 Jan 2026 17:06:21 -0500
Subject: [PATCH] Update v8_build.py

---
 builder/v8_build.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/builder/v8_build.py b/builder/v8_build.py
index e18696d..1b30d2e 100644
--- a/builder/v8_build.py
+++ b/builder/v8_build.py
@@ -253,7 +253,7 @@ def run_build(build_dir: Path, args: Args) -> None:
         run(
             executable,
             "build/linux/sysroot_scripts/install-sysroot.py",
-            "--arch=" + target_cpu,
+            "--arch=ppc64el --rust_target_arch=powerpc64le",
             cwd=get_v8_path(),
         )
 
-- 
2.34.1.windows.1

